sdk/textanalytics/Azure.AI.TextAnalytics/src/RecognizePiiEntitiesOptions.cs

```diff
@@ -25,5 +27,10 @@ public RecognizePiiEntitiesOptions()
         /// For more information see <a href="https://aka.ms/tanerpii"/>.
         /// </summary>
         public PiiEntityDomainType DomainFilter { get; set; }
+
+        /// <summary>
+        /// Filters the response entities to entities that match the <see cref="PiiEntityCategory"/> specified.
+        /// </summary>
+        public IList<PiiEntityCategory> CategoriesFilter { get; internal set; } = new List<PiiEntityCategory>();
```

Comment:
this should be `set` ?

Reply:
In .NET collections shouldn't be settable by the user, but we do set it internally when the user passes the data, so that is why it is `internal set`
there is a discussion around input collections being settable but nothing concrete yet, so for now, following the guideline